src/main/resources/warnings.json

```diff
@@ -5058,8 +5058,8 @@
     "url": "https://jenkins.io/security/advisory/2019-07-11/#SECURITY-1177",
     "versions": [
       {
-        "lastVersion": "0.13",
-        "pattern": ".*"
+        "lastVersion": "0.14",
```

Comment:
```suggestion
        "lastVersion": "0.13",
```

Last version is the last _affected_ version, for messages like "affected versions up to and including X".

Reply:
Applied